src/main/webapp/app/exam/participate/exercises/exercise-overview-page/exam-exercise-overview-page.component.ts

```diff
@@ -58,24 +59,24 @@ export class ExamExerciseOverviewPageComponent extends ExamPageComponent impleme
      * @param item the item for which the exercise status should be calculated
      * @return the sync status of the exercise (whether the corresponding submission is saved on the server or not)
      */
-    setExerciseIconStatus(item: ExamExerciseOverviewItem): 'synced' | 'notSynced' {
+    setExerciseIconStatus(item: ExamExerciseOverviewItem): 'synced' | '' {
```

Comment:
What does the emtpy string mean in this context? Could we assign a meaningful string?

Reply:
I just wanted to apply only synced class based on the submission status but I refactored this method anyways and used the same approach that you suggested above (enum as a return value), because I need to return at least three different classes 👍